src/main/java/org/example/objects/MusicList.java

```diff
@@ -0,0 +1,14 @@
+package org.example.objects;
```

Comment:
@SebasBarra02 To ensure the functionality of this class and if more methods will create it here, you should create unit tests, so please review it.

Reply:
test units were added